Source/RenderStream/Private/RenderStreamProjectionPolicy.cpp

```diff
@@ -62,12 +62,43 @@ void FRenderStreamProjectionPolicy::StartScene(UWorld* World)
     check(IsInGameThread());
     check(World);
 
-
     Module = FRenderStreamModule::Get();
     check(Module);
 
     Module->LoadSchemas(*World);
 
+    // Get player controller
+    if (APlayerController* ExistingController = UGameplayStatics::GetPlayerControllerFromID(World, PlayerControllerID))
+    {
+        Controller = ExistingController;
+    }
+    else if (APlayerController* NewController = UGameplayStatics::CreatePlayer(World))
+    {
+        PlayerControllerID = UGameplayStatics::GetPlayerControllerID(NewController);
+        Controller = NewController;
+    }
+    else
+        UE_LOG(LogRenderStream, Warning, TEXT("Could not set new view target for capturing."));
+
+    Stream = Module->StreamPool->GetStream(GetViewportId());
```

Comment:
This seems out of place as the same call is done in ConfigureCapture. 

I'm guessing it's mostly here for the initial PopulateStreamPool call. It would be nice if we could find a better place for it as it should only be called once on startup. If not, I'd just leave it in ConfigureCapture.

Reply:
I'm not sure this matters too much. All it's doing is checking the stream doesn't already exist before populating the stream pool. In this case it probably won't exist yet, but I don't know if there are circumstances in which the stream might have been loaded already.